onnxruntime/core/providers/tensorrt/onnx_ctx_model_helper.cc

```diff
@@ -137,15 +135,90 @@ ONNX_NAMESPACE::ModelProto* CreateCtxNodeModel(const GraphViewer& graph_viewer,
   return model_proto.release();
 }
 
+/*
+ * Get "EP context node" model path
```

Comment:
we need to constrain this function to not allow absolute paths and ../path
i.e. we shouldn't traverse outside of the current working directory or the directory of the model.onnx file

Reply:
added